src/services/LogsSceneQueryRunner.ts

```diff
@@ -13,6 +13,6 @@ export class LogsSceneQueryRunner extends SceneQueryRunner {
 
     // @todo can we make runWithTimeRange protected? (https://github.com/grafana/scenes/pull/866)
     // Hack to call private method
-    this['runWithTimeRange'](timeRange);
+    this['runWithTimeRangeAndScopes'](timeRange);
```

Comment:
oofdah, I should have cleaned this up by now, created https://github.com/grafana/logs-drilldown/issues/1201 to track my shame